BackofficeBundle/Form/DataTransformer/ChoiceArrayToStringTransformer.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace OpenOrchestra\BackofficeBundle\Form\DataTransformer;
+
+use Symfony\Component\Form\DataTransformerInterface;
+
+/**
+ * Class ChoiceArrayToStringTransformer
+ */
+class ChoiceArrayToStringTransformer implements DataTransformerInterface
+{
+    /**
+     * @param array|string $dataChoice
+     *
+     * @return string
+     */
+    public function transform($dataChoice)
+    {
+        if (is_scalar($dataChoice)) {
+            return (string) $dataChoice;
+        } elseif (is_array($dataChoice) && $dataChoice !== array()) {
+            return current($dataChoice);
+        } else {
+            return "";
+        }
+    }
+
+    /**
+     * @param string $dataChoice
+     *
+     * @return string
+     */
+    public function reverseTransform($dataChoice)
+    {
+        return $dataChoice;
```

Comment:
Is this never used ? It should transform a string into an array but it does nothing.